lib/groups.js

```diff
@@ -139,12 +179,54 @@ module.exports.createGroup = function (groupName, assessmentId, userId, authnUse
 };
 
 module.exports.leaveGroup = function (assessmentId, userId, authnUserId, callback) {
+  const params = [assessmentId, userId, authnUserId];
+  sqldb.call('group_leave', params, function (err) {
+    if (ERR(err, callback)) return;
+    callback();
+  });
+};
+
+module.exports.getAssessmentLevelPermissions = function (assessmentId, userId, callback) {
   const params = {
     assessment_id: assessmentId,
     user_id: userId,
-    authn_user_id: authnUserId,
   };
-  sqldb.query(sql.leave_group, params, function (err) {
+  sqldb.query(sql.get_assessment_level_permissions, params, function (err, result) {
+    if (ERR(err, callback)) return;
+    callback(result.rows[0]);
```

Comment:
When using callbacks, the first argument should always be the error:

```suggestion
    callback(null, result.rows[0]);
```